### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Leopere**

### CODE REVIEW

**Comment by Leopere** [comment]
Looking to see if these should cover our needs or if we should tweak them a bit.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe add a note about blurring/censoring nudity.

Comment (Leopere) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Added change.

**Comment by Leopere** [comment]
I think that I'm just going to merge this at this stage (18 days after I posted this) and if we want to revise it we can in subsequent PR's at this point.

**Pull request merged**